roxFile/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@eformservice/roxfile",
+  "servicename": "roxFile",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "cross-env HOTRELOAD=no DEBUG=bpmn-engine:123* ts-mocha -p tsconfig.json ./**/*.roxtest.ts --exit --recursive --timeout 40000 --require regenerator-runtime",
+    "copyproduction": "cross-var copyfiles package.json production && cd ./production && npm install --production && cd .. && cross-var copyfiles -u 1 production/node_modules/** $npm_package_servicename && rimraf production/",
+    "copyfiles": "cross-var copyfiles service.json dist/* $npm_package_servicename && npm run copyproduction",
+    "zip": "cross-var bestzip $npm_package_servicename.zip $npm_package_servicename",
+    "copyandzip": "cross-var npm run copyfiles && npm run zip && cross-var rimraf $npm_package_servicename"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/json-query": "^2.2.0",
+    "json-query": "^2.2.2",
+    "processhub-sdk": "https://github.com/roXtra/processhub-sdk/releases/download/v8.9.0-2/release.tgz",
```

Comment:
Please link this against SDK v8.10.0-0 now. The buildscript also automatically updates this version but for clarity I would prefer to also commit it with 8.10.0-0 since all other services are also committed with that version.